pgjdbc/src/main/java/org/postgresql/core/NativeQuery.java

```diff
@@ -76,4 +78,8 @@ public String toString(ParameterList parameters) {
   public static String bindName(int index) {
     return index < BIND_NAMES.length ? BIND_NAMES[index] : "$" + index;
   }
+
+  public static int bindsCount() {
```

Comment:
niggly little english grammar complaint here. bindCount is more correct or boundCount